components/megamenu/DesktopMenu/desktop-menu.tsx

```diff
@@ -13,6 +13,8 @@ export interface DesktopMenuProps {
   menuBarItems: NavMenuItem[];
 }
 
+export const ClosePopoverContext = createContext(null);
```

Comment:
Can you move this context to the context folder?

Reply:
Given that we've planned to relocate our megamenu component into a standalone package, I refrained from incorporating it under the context folder. What are your thoughts on this, @Harry-Ross?

Reply:
Fair enough - probably a good call then